app/assets/javascripts/home.js

```diff
@@ -46,14 +46,75 @@ $(document).ready(function(){
     $(this).find(".search_init").addClass("projects_filter");
   });
 
-  $("#installations_table").dataTable();
+  $("#installations_table").dataTable({
+    "iDisplayLength": 50
+  }).columnFilter({
+    aoColumns: [
+    {
+      type: "text",
+    },
+    {
+      type: "text",
+    },
+    {
+      type: "text",
+    },
+    {
+      type: "select",
+      values: ["Web", "Database", "Web & Database"]
```

Comment:
perhaps we could remove that hardcoding by using the gon gem + populating the lists from actual values in db? applies to both role and stage